README.md

```diff
@@ -42,19 +42,46 @@ Thus, `window.React` is then available to the Chrome React devtools extension.
 Alternately, you can set this in your config file:
 
 ```
+// webpack 1.x usage
 module: {
   loaders: [
     { test: require.resolve("react"), loader: "expose-loader?React" }
   ]
 }
+
+// webpack 2.x usage
+module: {
+  rules: [{
+          test: require.resolve('react'),
+          use: [{
+              loader: 'expose-loader',
+              options: 'React'
+          }]
+      }]
+}
 ```
 Also for multiple expose you can use `!` in loader string:
 ```
+// webpack 1.x usage
 module: {
   loaders: [
     { test: require.resolve("jquery"), loader: "expose-loader?$!expose-loader?jQuery" },
   ]
 }
+
+// webpack 2.x usage
+module: {
+  rules: [{
+          test: require.resolve('react'),
```

Comment:
since the webpack 1 example for multiple expose uses jQuery, why not have this one do it too for the sake of consistency?